linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator

```diff
@@ -24,49 +24,60 @@ properties ([
             choices: 'Ubuntu_16.04.3\nCentOS_7.4',
             name: 'DISTRO_VERSION',
             description: 'Distro version'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: 'Boot\nFunctional\nPerformance',
+            name: 'BISECT_TYPE',
+            description: 'Type of bisect that will be performed.'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: 'KVP\nCore\nFCOPY\nNET\nNET6\nProd_Checkpoint\nSTOR_VHDX\nVHDX_RESIZE\nLIVE_MIGR16\nLIVE_MIGR12R2',
+            name: 'FUNC_TEST_SUITE',
+            description: '[FUNC] the test suite which you want to run.'],
+        [$class: 'StringParameterDefinition',
+            name: 'FUNC_XML_PATH',
+            defaultValue: '',
+            description: '[FUNC] path to custom xml'],
```

Comment:
An absolute path to LISAcustom xml test. This path must a valid SMB share path, accessible by any test node.